src/modules/70-pipeline/components/PipelineStudio/StepPalette/StepPopover/StepPopover.tsx

```diff
@@ -47,6 +53,13 @@ function TooltipContent({ description, stepsFactory, stepData }: StepTooltipCont
         <Text font={{ size: 'small' }} color={Color.GREY_50}>
           {getString(description)}
         </Text>
+        {additionalInfo && (
+          <Layout.Horizontal margin={{ top: 'small' }}>
+            <Text font={{ size: 'small', weight: 'bold' }} color={Color.GREY_50}>
+              {getString(additionalInfo)}
+            </Text>
```

Comment:
Instead of using a specific size and weight, [is there a variation](https://uicore.harness.io/?path=/story/components-text--basic) that can be used?
```suggestion
            <Text font={{ variation: FontVariation.SMALL_BOLD }} color={Color.GREY_50}>
              {getString(additionalInfo)}
            </Text>
```